frontend/src/AlertConfiguration/AlertConfigurationComponents/ControlPanel.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, Fragment } from 'react';
```

Comment:
again, this file seems to look very similar to `ConfigForm.js` and `ConfigCard.js` and `ConfigList.js`...

would love to know the reason behind the similarities :)